packages/govern-core/contracts/OptimisticQueue.sol

```diff
@@ -183,6 +194,8 @@ contract OptimisticQueue is ERC3000, IArbitrable, MiniACL {
         );
         disputeItemCache[arbitrator][_disputeId] = bytes32(0); // refund gas, no longer needed in state
 
+        // Note: why not settling
```

Comment:
bc challenges here will only be made on fairly clear things (did a vote approving this pass?) so we can assume they are attacks and we want to take all their collateral